pkg/reconciler/v1alpha1/route/resources/cluster_ingress.go

```diff
@@ -52,6 +58,7 @@ func MakeClusterIngress(r *servingv1alpha1.Route, tc *traffic.TrafficConfig) *v1
 		},
 		Spec: makeClusterIngressSpec(r, tc.Targets),
 	}
+	return ci
```

Comment:
Why not keep the original code? seems like we don't need an extra variable `ci` here.